contracts/src/v0.8/keystone/CapabilitiesRegistry.sol

```diff
@@ -564,6 +575,9 @@ contract CapabilitiesRegistry is OwnerIsCreator, TypeAndVersionInterface {
       bytes32[] memory capabilityIds = node.hashedCapabilityIds;
       if (capabilityIds.length == 0) revert InvalidNodeCapabilities(capabilityIds);
 
+      bytes memory encryptionPublicKey = node.encryptionPublicKey;
+      if (encryptionPublicKey.length == 0) revert InvalidEncryptionPublicKey(encryptionPublicKey);
```

Comment:
cc @krehermann as I know you're working on prod setup atm

Reply:
@KuphJr Could you also update this script: https://github.com/smartcontractkit/chainlink/blob/1b41e69cca2f4622d367ef18733c36fcae433505/core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go#L4 so that it accurately reflects what we need on staging.